run-tests.pl

```diff
@@ -758,7 +758,7 @@ sub _run_test0
 
       Future->wait_any(
          $f_setup,
-         $loop->delay_future( after => 60 )
+         $loop->delay_future( after => 300 )
```

Comment:
300 seconds is a bit of a jump from 60. Perhaps we could start with x2 = 120?

What is the maximum amount of time a test takes on your machine?

Reply:
How do I measure it? TAP doesn't seem to print start/end time for successful tests by default.

Actually, there's no upper limit, as the machine is loaded with other tasks and may easily go into the thrashing state.

Reply:
Experiments with #996 show that with ×2 scaling, there still are 73 tests timing out; with ×3 and more, there still remain about 22–24 tests timing out, which all seem to be expected to fail because of timeouts though.